massgov/mayflower
# React/DP-12414: Responsive Table.

## Description
This updates the Table component for it to support responsiveness. 

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-12414)

## Steps to Test
1. Pull the branch and browse to atoms -> table-> table. Verify the Table works as intended for the ticket.

## Screenshots
![screencapture-localhost-6006-iframe-html-2019-02-01-12_23_13](https://user-images.githubusercontent.com/5257660/52138810-416b4680-261c-11e9-82c2-47961abaec29.png)
